### PR TITLE
Old Forms vs. Railsy Forms

### PR DESCRIPTION
**Pull request by kubilaycaglayan**
Dear TSE,

[README](https://github.com/mariosknl/Bare-Metal-Forms-and-Helpers/blob/re-former/README.md)

### Model:
- **User**
with columns: **username**, **email**, **password**

### Form Types:
- **HTML**
- **form_tag**
- **form_for**

### Extras:
- **Index page**: `localhost:3000/`
- **Error messages**
- `Red bordering style` added for the inputs that returns validation error

With all our respect...

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Rails.application.routes.draw do
+  get 'users/new'
+  get 'users/create'
+  get 'user/edit'
+  get 'user/update'
```
Comment (suretrust) [inline config/routes.rb:5]: You don't need these anymore.

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<h1>All <%= @users.all.count %> Users</h1>
+
+<ol>
+<% @users.all.each do |user| %>
+  <li><%= link_to user.username, edit_user_path(user.id) %></li>
+<% end %>
+</ol>
+
+<%= link_to 'Create New User', new_user_path %>
```
Comment (suretrust) [inline app/views/users/index.html.erb:9]: According to the routes, you are not using this at all.

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def index
+    @users = User.all
+  end
```
Comment (suretrust) [inline app/controllers/users_controller.rb:8]: You don't need the index action in this exercise.

File: app/views/users/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  <label for="password">Password: </label>
+  <input type="password" name="user[password]"><br /><br />
+  <input type="submit">
+</form>
+
+<%= form_tag '/users'%>
+  <%= label_tag 'username' %>
+  <%= text_field_tag 'username', nil, placeholder: 'similar to David' %><br /><br />
+  <%= label_tag 'email' %>
+  <%= text_field_tag 'email', nil, placeholder: 'example@sm.com' %><br /><br />
```
Comment (suretrust) [inline app/views/users/new.html.erb:20]: This should be `email_field_tag`.

File: app/controllers/users_controller.rb (hunk 3 above)
Comment (kubilaycaglayan) [reply app/controllers/users_controller.rb:8]: We wanted to add an extra feature. We want to keep it if it is possible.

File: app/views/users/new.html.erb (hunk 4 above)
Comment (kubilaycaglayan) [reply app/views/users/new.html.erb]: Corrected thank you very much.

**Comment by kubilaycaglayan** [comment]
Dear TSE,

1. **Routes** reconfigured.
2. We know that **the index page** is not needed on this project, we wanted to provide an extra feature.
3. **email_field_tag** corrected

With all our respect...

File: config/routes.rb (hunk 1 above)
Comment (kubilaycaglayan) [reply config/routes.rb]: Corrected.

**Pull request merged**